#!/usr/bin/python3
def raise_exception_msg(message=""):  # raise with msg
    raise NameError(message)
